### DISCUSSION TITLE
Disable floating window movement?

### DISCUSSION THREAD

**Discussion by avalonv** [koreader/koreader#9646] [Q&A] [answered]
Is it possible to disable the ability to move floating/context windows? I find that I never use that feature and it ends up causing me more issues when I have to try to recenter the window than it solves...

**Comment by poire-z** [comment, marked as answer]
You could help yourself with a user patch: https://github.com/koreader/koreader/wiki/User-patches

Previously made ones and how to use them among https://github.com/koreader/koreader/issues?q=sort%3Aupdated-desc+label%3A%22User+patch+available%22.

Here's one quickly made that seems to work with InfoMessages and DictLookup (may be I missed some other ones that have some other pecularities):
[2-movable-not-movable.lua.txt](https://github.com/koreader/koreader/files/9854501/2-movable-not-movable.lua.txt)

```lua
local MovableContainer = require("ui/widget/container/movablecontainer")

MovableContainer.init = function(self)
    self.ignore_events = {"touch", "swipe", "hold", "hold_pan", "hold_release", "pan", "pan_release"}
end
local nope = function() return false end
MovableContainer.onMovableSwipe = nope
MovableContainer.onMovableHold = nope
MovableContainer.onMovableHoldPan = nope
MovableContainer.onMovableHoldRelease = nope
MovableContainer.onMovableTouch = nope
MovableContainer.onMovablePan = nope
MovableContainer.onMovablePanRelease = nope
```

**Reply by ghost** [reply]
FWIW: I have added this to https://github.com/koreader/koreader/wiki/User-patches

**Reply by avalonv** [reply]
Thank you very much, this is super helpful.
